fenix/app/src/main/res/values/strings.xml

```diff
@@ -1307,12 +1307,16 @@
     <!-- Subtitle for the tabs item in Delete browsing data, parameter will be replaced with the number of open tabs -->
     <string name="preferences_delete_browsing_data_tabs_subtitle">%d tabs</string>
     <!-- Title for the data and history items in Delete browsing data -->
-    <string name="preferences_delete_browsing_data_browsing_data_title">Browsing history and site data</string>
+    <string name="preferences_delete_browsing_data_browsing_data_title" moz:removedIn="118">Browsing history and site data</string>
```

Comment:
@iTrooz @MatthewTighe I am wondering if this string is still being used? Usually, we would also append `tools:ignore="UnusedResources"` and ensure the strings aren't being used anywhere when removing them.

Reply:
It is not supposed to be used anymore, but after verifying, it is indeed used at https://github.com/mozilla-mobile/firefox-android/blob/f931d36eec654a833174d0fdb451713ae2d63b43/fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuDeleteBrowsingDataRobot.kt#L181. (And line 189 for the `preferences_delete_browsing_data_cookies` string)

This is an oversight on my part, sorry about that.
Do you want me to submit a follow-up PR to fix this ?

Note: I noticed this (testing ?) code is called by the function https://github.com/mozilla-mobile/firefox-android/blob/f931d36eec654a833174d0fdb451713ae2d63b43/fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuDeleteBrowsingDataRobot.kt#L34, which is unused. I am not familiar with the codebase, but maybe this a problem ?

Reply:
@iTrooz If you could investigate a follow clean up, that would be greatly appreciated. I think what we hope to accomplish here is to clean up any usage of this string at minimum. If it makes sense to adjust this `assertAllOptionsAndCheckBoxes`, then we should also do update the test to match what we are looking for in the settings screen.

@AndiAJ Could you advise about these unused functions that still persist in the code? Would it be worthwhile to keep it or to remove it since they are unused?

Reply:
Hi @gabrielluong , @iTrooz , just checked and yes, those unused functions can be deleted.

Reply:
Hey, this is done: https://github.com/iTrooz/firefox-android/tree/fixup

How should I go about making my PR ? I need to put a bug ID in it. Do I re-use this one (1813788), or do I create a bug on the bugzilla, specifically for this PR ?

Reply:
@iTrooz Feel free to reuse the same Bug number 1813788. I would suggest is to do something like append a "Part 2" such _"Bug 1813788 - Part 2: \<Description\>"_.  That way anyone looking at the history would know there's an existing "Part 1" which would contain your core changes for the bug.